### DISCUSSION TITLE
Why are some definitions described by the `colour.describe_conversion_path` definition seemingly missing?

### DISCUSSION THREAD

**Discussion by blue-j** [colour-science/colour#1154] [General]
**CAM16_to_JMh_CAM16**
**CIECAM16_to_JMh_CIECAM16**
**CIECAM02_to_JMh_CIECAM02**
**Hellwig2022_to_JMh_Hellwig2022**

do not appear to exist in the library, but are referenced in the Automatic Conversion Graph and `colour.describe_conversion_path`

Am I missing something?

- J

**Comment by thomasmansencal** [comment]
Hello,

See my reply here: https://github.com/colour-science/colour/issues/1155#issuecomment-1556267113
